src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -359,6 +359,10 @@ export default {
 }
 
 .loan-categories {
+	& .row {
+		max-width: 69.15rem;
```

Comment:
This feels hacky, but the design given is wider than our container and I don't see any other way to get 3 cards in the row